### ISSUE TITLE
Gitpod might need updating

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_project#163]
Just did a fresh install on Gitpod, from the readme (https://gitpod.io/#https://github.com/localgovdrupal/localgov_project) and drush was not available, and php was not available,

Gonna try updating to the latest Drupalpod image: https://hub.docker.com/layers/drupalpod/drupalpod-gitpod-base/20240419/images/sha256-f2faea266bd29c21d84d00d17ad5a9effe23c920f82bea23d5395ec6d082d8c3?context=explore

**Comment by markconroy** [comment]
Should we remove GitPod stuff from our repo. I don't think GitPod exists any more.

**Comment by andybroomfield** [comment]
Yeah, Gitpod has now become an AI code assistant thing and Gitpod classic has been discontinued.
Github code spaces might be an option, or we could check if Drupal.org will have a similar thing or not.

What was the original motivation? As a way of quickly trying LGD? Maybe the [Drupal forge link](https://www.drupalforge.org/template/local-gov-drupal) is better?
